pages/services/dse/3.0.0-6.0.1/limitations/index.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: layout.pug
+navigationTitle:
+excerpt:
+title: Limitations
+menuWeight: 100
+model: /services/dse/data.yml
+render: mustache
+---
+
+#include /services/include/limitations.tmpl
+#include /services/include/limitations-zones.tmpl
+
+## Service Limits
+- Multiple DSE Instances on a host is not supported in production.
+- Stopping or restarting a DSE node from OpsCenter is not supported. Use `dcos pod restart` to restart DSE nodes from the DC/OS CLI.
```

Comment:
Some of this is no longer correct:

* You can restart DSE nodes in OpsCenter
* DSE tasks and opscenter tasks are independent now, so if one goes down the other will not.  The statement about the DSE node restarting if the agent crashes, is not accurate anymore.
* Point-in-time restore functionality through OpsCenter IS working and supported now.

Reply:
Thanks for mentioning this and as it is no longer in order, I have removed these points from the guide.
Please suggest me if it further require anything to be changed/removed.